Sources/BraintreeUIKit/Components/BTUIKFormField.m

```diff
@@ -36,6 +36,7 @@ - (instancetype)initWithFrame:(CGRect)frame {
         self.textField.adjustsFontSizeToFitWidth = YES;
         self.textField.returnKeyType = UIReturnKeyNext;
         self.textField.keyboardAppearance = [BTUIKAppearance sharedInstance].keyboardAppearance;
+        self.textField.textAlignment = [BTUIKViewUtil naturalTextAlignment];
```

Comment:
I changed this to default to natural text alignment, which is what we always want if the text field doesn't have a label. When label text is set, there's logic to determine the correct text alignment for the text field, depending on the font size.